worksheet_B/README.md

```diff
@@ -0,0 +1,21 @@
+PSEUDOCODE:
```

Comment:
The worksheet asked you to provide pseudocode for the calculation of the "similarity score", i.e. comparing the guessed word to the secret word and showing the number of matching letters.